.github/release-please/manifest.json

```
@@ -0,0 +1,3 @@
+{
+  ".": "0.2.4"
```

We released 0.2.5 yesterday so I assume this needs to be updated now:
```suggestion
  ".": "0.2.5"
```

---

Sure.